app/src/main/res/values-ja/strings.xml

```diff
@@ -367,7 +367,7 @@
   <string name="storage_permission">ストレージの許可</string>
   <string name="write_storage_permission_rationale_for_image_share">画像の投稿には、外部ストレージへのアクセス権限を許可してください。</string>
   <string name="nearby_notification_dismiss_message">今後、画像募集中の近くの場所は表示されません。ただしこの通知機能は、いつでも「設定」画面から有効にできます。</string>
-  <string name="step_count">全部で%2$d件のうち%1$d件目の手順：%3$s</string>
```

Comment:
also, we never edit string files of languages manually. We only edit the default strings file and translate updates handles the rest. So this problem should be solved without ja/strings.xlm is edited.

Reply:
@neslihanturan Thanks for reviewing my changes. I will resolve it ASAP.

Reply:
@neslihanturan Please once review the changes